_data/versions.yml

```diff
@@ -1,7 +1,192 @@
 v3.0:
+- title: v3.0.1
+  note: |
+    22 December 2017
+
+    #### What's new
+    
+    ##### Breaking changes
+    
+    This release includes breaking changes to the data and resource model. [Review the documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) 
+    carefully before attempting an upgrade. Some highlights include: 
+    
+    - You must upgrade to Calico v2.6.5 before you can upgrade to v3.0.1.
+    
+    - Those accessing the etcd datastore directly must complete a one-time migration.
+    
+    - You must convert any customized manifests via `calicoctl convert` before you can
```

Comment:
This is incorrect, this could be interpreted as Kubernetes manifests too.  Only Calico manifests, that use the Calico resource models, should be converted.

Reply:
OK thanks, good info.